TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PinpointDrive.java

```diff
@@ -32,8 +34,8 @@ the tracking point the Y (strafe) odometry pod is: forward of the center is a po
          */
         //These are tuned for 3110-0002-0001 Product Insight #1
         // RR localizer note: These units are inches, presets are converted from mm (which is why they are inexact)
-        public double xOffset = -6.9469116647256905;
-        public double yOffset = -6.260357700406137; //1 * (TRACK_WIDTH / 2) - 2.25;
+        public double xOffset = -7.004628305498452;
+        public double yOffset =  -6.048438856096477; //1 * (TRACK_WIDTH / 2) - 2.25;
```

Comment:
This was while tuning the competition robot.  TODO - find a way to swap those out